packs/actions/disarming-interception.json

```diff
@@ -11,14 +11,30 @@
         },
         "category": "defensive",
         "description": {
-            "value": "<p><strong>Requirements</strong> You have your fist positioned to parry</p>\n<p><strong>Trigger</strong> An enemy within your reach targets you or an ally with a weapon Strike</p><hr /><p><strong>Effect</strong> s You attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Disarm] the weapon the enemy is attacking with. You gain a +2 status bonus to this Disarm check, and if the check is successful, the triggering attack is disrupted. If the Disarm attempt is a critical success and you have a hand free, you can catch the disarmed weapon in your hand instead of it falling to the ground in the target's space.</p>"
+            "value": "<p><strong>Requirements</strong> You have your fist positioned to parry</p>\n<p><strong>Trigger</strong> An enemy within your reach targets you or an ally with a weapon Strike</p><hr /><p><strong>Effects</strong> You attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Disarm] the weapon the enemy is attacking with. You gain a +2 status bonus to this Disarm check, and if the check is successful, the triggering attack is disrupted. If the Disarm attempt is a critical success and you have a hand free, you can catch the disarmed weapon in your hand instead of it falling to the ground in the target's space.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Requirements</strong> You have your fist positioned to parry</p>\n<p><strong>Trigger</strong> An enemy within your reach targets you or an ally with a weapon Strike</p><hr /><p><strong>Effect</strong> You attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Disarm] the weapon the enemy is attacking with. You gain a +2 status bonus to this Disarm check, and if the check is successful, the triggering attack is disrupted. If the Disarm attempt is a critical success and you have a hand free, you can catch the disarmed weapon in your hand instead of it falling to the ground in the target's space.</p>"
```